index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+	<head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<title>Hello Microverse</title>
+        <link rel="stylesheet" href="styles.css">
+	</head>
+	<p class="text">"Hello Microverse!"</p>
```

Comment:
- [ ]  Kindly use an h1 tag instead of a p tag to acoplish with the project requirements: